src/messaging/messaging.ts

```diff
@@ -250,17 +251,16 @@ export class Messaging {
       });
   }
 
-  // TODO: Update the comment based on the implementation
   /**
   * Sends each message in the given array via Firebase Cloud Messaging.
   *
-  * Unlike {@link Messaging.sendAll}, this method makes a single RPC call for each message in the given array.
+  * Unlike {@link Messaging.sendAll}, this method makes a single RPC call for each message
```

Comment:
Should this be a RPC call or an HTTP call?

Reply:
I think we use RPC in other places in the SDK. Let's get the docs reviewed so we can address all these naming issues. Thanks!

Reply:
Sounds good! Based on the implementation, I think we actually send an HTTP call instead of an internal RPC call. I can leave this as it is now.